justfile

```diff
@@ -14,10 +14,13 @@ check:
 
 # Run all the tests.
 test language="[rust|python]" : (_run_lang language \
-        "HUGR_TEST_SCHEMA=\"1\" cargo test --all-features" \
+        "HUGR_TEST_SCHEMA=\"1\" cargo test --features extension_inference" \
         "poetry run pytest"
     )
 
+proptest:
+    cargo test --all-features
```

Comment:
R u sure you want to turn `extension_inference` on here too? ;-)

I guess this makes sense but I'm not 100% sure about the naming